stage0/layout.ld

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2022 The Project Oak Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+HIDDEN(TOP = 4096M);
+HIDDEN(BIOS_SIZE = 256K);
```

Comment:
Is this intended to be different to the 64KiB specified in the readme?

Reply:
Good catch, no. Once Rust came into play 64K wasn't enough any longer.